back.py

```diff
@@ -38,6 +44,24 @@ def play(file):
     soundPlayStop = True
     soundPlaying = False
 
+def call(number):
+    client = Client(account_sid, auth_token)
+    call = client.calls.create(
+                        url='http://demo.twilio.com/docs/voice.xml',
+                        to=number,
+                        from_=source_phone_number
+                    )
+
+def sendSMS(number, message):
+    client = Client(account_sid, auth_token)
```

Comment:
is the client can be initialised in the beginning?

Reply:
yes, but it is not very expensive to create this client, will leave it here.